lib/dialects/mssql/query-generator.js

```diff
@@ -492,16 +492,21 @@ class MSSQLQueryGenerator extends AbstractQueryGenerator {
 
     let whereClause = this.getWhereConditions(where, null, model, options);
     let limit = '';
+    let order = '';
 
     if (options.limit) {
       limit = ` TOP(${this.escape(options.limit)})`;
     }
 
+    if (options.order) {
+      order = ` ORDER BY ${options.order.map(x => x.join(' ')).join(', ')}`;
```

Comment:
This needs escaping as well, but I would like to just use internal helpers that are used to build order statement for other queries